coderd/coderdtest/oidctest/idp.go

```diff
@@ -794,6 +794,7 @@ func (f *FakeIDP) newToken(t testing.TB, email string, expires time.Time) string
 func (f *FakeIDP) newRefreshTokens(email string) string {
 	refreshToken := uuid.NewString()
 	f.refreshTokens.Store(refreshToken, email)
+	f.logger.Info(context.Background(), "new refresh token", slog.F("email", email), slog.F("token", refreshToken))
```

Comment:
I'm guessing this was for debugging some other tests?

The tests written did not require the oidctest.

Regardless, I don't mind more logs in this code

Reply:
Yeah, this test was failing on an early revision of the change. I added the logs to diagnose, and left them in because they seemed useful.

Reply:
:+1: